test/integration/util/authentication.go

```diff
@@ -516,6 +516,21 @@ func PrepareSpokeAgentNamespace(kubeClient kubernetes.Interface, namespace strin
 }
 
 func GetFilledHubKubeConfigSecret(kubeClient kubernetes.Interface, secretNamespace, secretName string) (*corev1.Secret, error) {
+	secret, err := GetFilledAWSHubKubeConfigSecret(kubeClient, secretNamespace, secretName)
+	if err != nil {
+		return nil, err
+	}
+	if _, existed := secret.Data["tls.crt"]; !existed {
+		return nil, fmt.Errorf("tls.crt is not found")
+	}
+
+	if _, existed := secret.Data["tls.key"]; !existed {
+		return nil, fmt.Errorf("tls.key is not found")
+	}
+	return secret, nil
+}
+
+func GetFilledAWSHubKubeConfigSecret(kubeClient kubernetes.Interface, secretNamespace, secretName string) (*corev1.Secret, error) {
```

Comment:
if it is only to check the hubkubeconfig key, you might want to call it GetHubKubeConfigFromSecret